tpgtools/templates/test_file.go.tmpl

```diff
@@ -46,6 +46,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 
 	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
```

Comment:
I might just be missing it, but why do we need this import here? I don't see the package being used.

Reply:
Thanks for reviewing the PR.

The package `envvar` is used to create the context map by calling the function `s.ExpandContext`.
https://github.com/GoogleCloudPlatform/magic-modules/blob/2422055fe1c63c36c942b89a029471bba3a263dc/tpgtools/templates/test_file.go.tmpl#L58

`ExpandContext` function uses the `translationMap` which needs the envvar package.
https://github.com/GoogleCloudPlatform/magic-modules/blob/2422055fe1c63c36c942b89a029471bba3a263dc/tpgtools/sample.go#L364

Reply:
Interesting, thanks for explaining!